### PR TITLE
feat: add reusable action to deploy job to kubernetes cluster on eks

### PR DESCRIPTION
**Pull request by thesmo**

### CODE REVIEW

File: .github/workflows/kubectl-deploy-job-eks.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+name: Superlinter
```
Comment (Wagan8r) [inline .github/workflows/kubectl-deploy-job-eks.yml:1]: I imagine you want a different name here.

File: .github/workflows/kubectl-deploy-job-eks.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        type: string
+      job_file:
+        description: 'filename of the job'
+        default: 'job.yaml'
+        required: true
+        type: string
+      tag:
+        description: 'Image tag to deploy'
+        default: 'latest'
+        required: true
```
Comment (Wagan8r) [inline .github/workflows/kubectl-deploy-job-eks.yml:44]:
```suggestion
        required: false
```
Same comment

File: .github/workflows/kubectl-deploy-job-eks.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+          theargs: ${{ inputs.args }}
+          jobname: ${{ inputs.job_name }}
+          jobfile: ${{ inputs.job_file }}
+          newfile: ${{ inputs.job_name }}-new.yml
+        run: |
+          cp ${jobfile} ${newfile}
+          sed -i "s|image:.*|image: ${registry}:${tag}|g" ${newfile}
+          sed -i "s|command:.*|command: ${thecmd}|g" ${newfile}
+          sed -i "s|args:.*|args: ${theargs}|g" ${newfile}
+          echo && echo
```
Comment (Wagan8r) [inline .github/workflows/kubectl-deploy-job-eks.yml:87]: `echo && echo`?? What does that do?

**Review by Wagan8r — APPROVED** [review]
Just some minor typos, otherwise it looks good to me

File: .github/workflows/kubectl-deploy-job-eks.yml (hunk 1 above)
Comment (thesmo) [reply .github/workflows/kubectl-deploy-job-eks.yml:1]: You're right

File: .github/workflows/kubectl-deploy-job-eks.yml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        required: true
+        type: string
+      job_name:
+        description: 'name of job'
+        required: true
+        type: string
+      job_file:
+        description: 'filename of the job'
+        default: 'job.yaml'
+        required: true
```
[the remark this answers is not in the document]
Comment (thesmo) [reply .github/workflows/kubectl-deploy-job-eks.yml:39]: Makes sense

File: .github/workflows/kubectl-deploy-job-eks.yml (hunk 3 above)
Comment (thesmo) [reply .github/workflows/kubectl-deploy-job-eks.yml:87]: Adds 2 new lines. Not needed. Will remove.

File: .github/workflows/kubectl-deploy-job-eks.yml (hunk 2 above)
Comment (thesmo) [reply .github/workflows/kubectl-deploy-job-eks.yml:44]: Going to remove the default on this one.

**Pull request merged**